app/main.py

```diff
@@ -2,0 +1,22 @@
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    mode, source, destination = command.split()
+
+    if mode != "mv" or source == destination:
+        raise ValueError("use: mv source destination")
+
+    if not os.path.exists(source):
+        raise FileNotFoundError(f"File {source} not found")
+
+    if destination.endswith("/"):
```

Comment:
Using 'os.path.join' here is a good practice as it ensures cross-platform compatibility.